src/main/c/Jep/pyembed.c

```diff
@@ -256,57 +300,48 @@ void pyembed_preinit(JNIEnv *env,
  */
 static int pyjtypes_ready(void)
 {
-    if (PyType_Ready(&PyJObject_Type) < 0) {
+    // start at the top with object
+    if (jep_jobject_type_ready() < 0) {
```

Comment:
I actually think this heap-type cleanup is a good idea even if we don't take this PR.